Project2/style.css

```
@@ -0,0 +1,115 @@
+body {
+    box-sizing: border-box;
+    height: auto;
+    width: 640px;
+}
+
+.mainimage {
+    background-image: url(Images/TheFavouriteWebCardImage.jpg);
+    background-size: cover;
+    height: 380px;
+    position: relative;
+    width: 640px;
+    z-index: 3;
+
+
+}
+
+
+
+.content {
+    background: linear-gradient(to left, #42513F, transparent 190%);
```

I totally understand how you landed on `190%`, but it's a little hard to wrap you're head around since it requires thinking of the gradient expanding _beyond_ its container. Something like this might get you closer to the effect you want without stretching beyond the container:
```suggestion
    background: linear-gradient(to left, hsla(110, 13%, 28%, 0.95), hsla(110, 13%, 28%, 0.75), hsla(110, 13%, 28%, 0.4) 100%);
```
This takes 3 different transparency levels of your basic color (#42513F) and applies them at 3 break points -- 95% opaque at the far right, 75% opaque at the middle and 40% at the far left.

You can see more possibilities in the [MDN docs on `linear-gradient`](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient).